### PR TITLE
Documenting how to use Beam twitter handle

### PR DESCRIPTION
**Pull request by aijamalnk**
Adding a short documentation to Community page about using Beam twitter.
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python2/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python2_PVR_Flink_Cron/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)
XLang | --- | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_XVR_Flink/lastCompletedBuild/) | --- | --- | ---

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PreCommit_PythonLint_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by aijamalnk** [comment]
Staged page:
http://apache-beam-website-pull-requests.storage.googleapis.com/10082/community/twitter-handle/index.html

**Comment by aijamalnk** [comment]
r: @tweise @matthiasa4 PTAL : )

File: website/src/community/twitter-handle.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Beam Twitter handle 
+## What is it and what are the goals?
+Apache Beam community thrives to be inclusive to everyone. As part of this effort, we want to enable any community member to share anything interesting and fun related to Beam via its [official twitter handle](https://twitter.com/ApacheBeam).
```
Comment (tweise) [inline website/src/community/twitter-handle.md]:
```suggestion
Apache Beam community thrives to be inclusive to everyone. As part of this effort, we want to enable any community member to share anything interesting and fun related to Beam via its [official Twitter handle](https://twitter.com/ApacheBeam).
```

File: website/src/community/twitter-handle.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+The Twitter feed found here is owned by the Apache Beam PMC, and the process of adding new tweets into the feed is documented below.
```
Comment (tweise) [inline website/src/community/twitter-handle.md]:
```suggestion
The Twitter feed found here is owned by the Apache Beam PMC, and the process of proposing new tweets is documented below.
```

File: website/src/community/twitter-handle.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- Go to [s.apache.org/beam-tweets](https://s.apache.org/beam-tweets) and request “Edit” access.
+- After you are able to edit the document, fill out the necessary fields:
+	- Date - the date when you are filling out the document
+	- Author - your name
+	- Topic - what is the tweet about
+	- Content - the actual text of the tweet or link to the tweet that need to be retweeted
+	- Links/media - anything you want to add to the tweet, eg. photos, videos, references
+	- Deadline - by when the tweet needs to be out, or preferred date when it needs to be out 
+- After you type in your tweets, PMC members that are subscribed to the document will get notified and review the content.
+- After the PMC approval, a PMC member with access to the twitter handle will send out the tweet.
```
Comment (tweise) [inline website/src/community/twitter-handle.md]:
```suggestion
- After approval, a PMC member with access to the Twitter handle will publish the tweet.
```

File: website/src/community/twitter-handle.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Beam Twitter handle 
+## What is it and what are the goals?
+Apache Beam community thrives to be inclusive to everyone. As part of this effort, we want to enable any community member to share anything interesting and fun related to Beam via its [official Twitter handle](https://twitter.com/ApacheBeam).
```
Comment (matthiasa4) [inline website/src/community/twitter-handle.md]: @aijamalnk Thanks for helping out on this - really glad this is part of the webpage now.

Am I misunderstanding this sentence or do you mean 'Apache Beam community tries...' or 'Apache Beam community thrives by being...'? :)

Comment (aijamalnk) [reply website/src/community/twitter-handle.md]: @matthiasa4 it is tries :) feel free to submit a PR with the fix :)

**Pull request merged**